ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -2363,6 +2363,62 @@ private void LoadExtensions(WorksheetExtensionList extensions, XLWorksheet ws)
                     xlConditionalFormat.Colors.Add(this.GetColor(color));
                 }
             }
+
+            foreach (var sparklineGroups in extensions
+                .Descendants<DocumentFormat.OpenXml.Office2010.Excel.SparklineGroups>())
+            {
+                foreach (var slg in sparklineGroups.Descendants<DocumentFormat.OpenXml.Office2010.Excel.SparklineGroup>())
+                {
+                    var sparklineGroup = ws.SparklineGroups.Add(ws);
+
+                    if (slg.AxisColor != null) sparklineGroup.AxisColor = ExtractColor(slg.AxisColor.Rgb.Value);
+                    if (slg.FirstMarkerColor != null) sparklineGroup.FirstMarkerColor = ExtractColor(slg.FirstMarkerColor.Rgb.Value);
+                    if (slg.LastMarkerColor != null) sparklineGroup.LastMarkerColor = ExtractColor(slg.LastMarkerColor.Rgb.Value);
+                    if (slg.HighMarkerColor != null) sparklineGroup.HighMarkerColor = ExtractColor(slg.HighMarkerColor.Rgb.Value);
+                    if (slg.LowMarkerColor != null) sparklineGroup.LowMarkerColor = ExtractColor(slg.LowMarkerColor.Rgb.Value);
+                    if (slg.SeriesColor != null) sparklineGroup.SeriesColor = ExtractColor(slg.SeriesColor.Rgb.Value);
+                    if (slg.NegativeColor != null) sparklineGroup.NegativeColor = ExtractColor(slg.NegativeColor.Rgb.Value);
+                    if (slg.MarkersColor != null) sparklineGroup.MarkersColor = ExtractColor(slg.MarkersColor.Rgb.Value);
+
+                    if (slg.Markers != null) sparklineGroup.Markers = slg.Markers;
+                    if (slg.High != null) sparklineGroup.High = slg.High;
+                    if (slg.Low != null) sparklineGroup.Low = slg.Low;                                        
+                    if (slg.First != null) sparklineGroup.Low = slg.First;
+                    if (slg.Last != null) sparklineGroup.Low = slg.Last;
```

Comment:
```
                    if (slg.Low != null) sparklineGroup.Low = slg.Low;                                        
                    if (slg.First != null) sparklineGroup.Low = slg.First;
                    if (slg.Last != null) sparklineGroup.Low = slg.Last;
```

That's why I like unit tests

Reply:
I wouldn't say I like them, but that's why they're a necessity.